### PR TITLE
[DECLUTTER BUILDING] separate structure, remove CodeCoverage

### PR DESCRIPTION
**Pull request by MaxHillebrand**
This ready for review branch changes the structure of `building-wasabi` to

```
### Contributing to Wasabi

- [Technical Overview of Wasabi Wallet](/building-wasabi/TechnicalOverview.md)
- [Contribution Checklist](/building-wasabi/ContributionChecklist.md)
- [Contribution Game](/building-wasabi/ContributionGame.md)
- [Dojo](/building-wasabi/Dojo.md)
- [ToDo List](/building-wasabi/ToDo.md)

### Advanced

- [False Positive](/building-wasabi/FalsePositive.md)
- [Security Policy](/building-wasabi/Security.md)
- [Hardware Wallet Integration Testing](/building-wasabi/HardwareWalletTestingGuide.md)
- [Developers debugging Guide](/building-wasabi/HowToDebug.md)
- [Ports](/building-wasabi/Ports.md)
- [MIT License](/building-wasabi/LICENSE.md)
- [Credits](/building-wasabi/Credits.md)
```

It also removes `CodeCoverage.md` as this is part of the [main repo](https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi.Documentation/CodeCoverage.md). 

Further, it streamlines across all pillars to have `## Chapters` and `### Sub Chapter`

The branch builds successfully.

### CODE REVIEW

File: docs/FAQ/README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Including about the basics of privacy and Bitcoin in general.
 The important details of how to install Wasabi properly and the best practices how to use it.
```
Comment (davterra) [inline docs/FAQ/README.md]:
```suggestion
The important details of how to install Wasabi properly and the best practices for using it.
```

File: docs/FAQ/README.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ Including about the basics of privacy and Bitcoin in general.
...
 Further the questions for those who are interested in contributing to Wasabi.
```
Comment (davterra) [inline docs/FAQ/README.md]:
```suggestion
Provide guidance for those who are interested in contributing to Wasabi.
```

File: docs/building-wasabi/README.md [hunk 3]
```diff
@@ -12,16 +12,20 @@ Here you find out about the Wasabikas who have already supported this open sourc
 Some of the issues they are working on now as well as the research for future ideas.
```
Comment (davterra) [inline docs/building-wasabi/README.md]:
```suggestion
You will also find some of the issues they are working on now, the research for future ideas, and all of the necessary information needed to quickly begin making your own invaluable contributions.
```

File: docs/building-wasabi/README.md [hunk 4]
```diff
@@ -12,16 +12,20 @@ Here you find out about the Wasabikas who have already supported this open sourc
...
 And all the necessary information you need to quickly start with your invaluable contribution.
```
Comment (davterra) [inline docs/building-wasabi/README.md]:
```suggestion

```

File: docs/FAQ/README.md (hunk 2 above)
Comment (MaxHillebrand) [reply docs/FAQ/README.md]:
```suggestion
Further providing guidance for those who are interested in contributing to Wasabi.
```

Comment (davterra) [reply docs/FAQ/README.md]: Sounds good. Glad you added the needed comma after "Further".

File: docs/FAQ/README.md [hunk 5]
```diff
@@ -9,8 +9,8 @@
 
 This is an archive with some of the many questions that Wasabikas have frequently.
```
Comment (davterra) [inline docs/FAQ/README.md]:
```suggestion
This is an archive with some of the many questions that Wasabikas have frequently asked, including:
```

File: docs/FAQ/README.md [hunk 6]
```diff
@@ -9,8 +9,8 @@
...
 Including about the basics of privacy and Bitcoin in general.
```
Comment (davterra) [inline docs/FAQ/README.md]:
```suggestion
- Questions about the basics of privacy, and Bitcoin in general.
```

File: docs/FAQ/README.md [hunk 7]
```diff
@@ -9,8 +9,8 @@
...
-The important details of how to install Wasabi properly and the best practices how to use it.
-Further the questions for those who are interested in contributing to Wasabi.
+The important details of how to install Wasabi properly and the best practices for using it.
```
Comment (davterra) [inline docs/FAQ/README.md]:
```suggestion
- The important details of how to install Wasabi properly and the best practices for using it.
```

File: docs/FAQ/README.md [hunk 8]
```diff
@@ -9,8 +9,8 @@
...
+Further, providing guidance for those who are interested in contributing to Wasabi.
```
Comment (davterra) [inline docs/FAQ/README.md]:
```suggestion
- Further guidance for those who are interested in contributing to Wasabi.
```

**Comment by MaxHillebrand** [comment]
Sorry @davterra , I was too fast in merging it... 
Can you please create a PR with a proper re-write?

**Comment by davterra** [comment]
Will do, thanks.

**Pull request merged**